js/src/pages/price-benchmark/index.js

```diff
@@ -9,6 +9,7 @@ import { Card } from '@wordpress/components';
  * Internal dependencies
  */
 import { glaData } from '~/constants';
+import ExperienceRatingBanner from '~/components/experience-rating-banner';
 import AppNotice from '~/components/app-notice';
 import AppSpinner from '~/components/app-spinner';
 import MainTabNav from '~/components/main-tab-nav';
```

Comment:
```suggestion
import AppNotice from '~/components/app-notice';
import AppSpinner from '~/components/app-spinner';
import ExperienceRatingBanner from '~/components/experience-rating-banner';
import MainTabNav from '~/components/main-tab-nav';
```

Nit-pick

Reply:
👍 Moved Banner as well since we are alpha sorting 😁